### ISSUE TITLE
Reproducibility on VOC12

### ISSUE THREAD

**Issue by arnike** [DrSleep/tensorflow-deeplab-resnet#197]
Hi Vladimir,

I was wondering about the training specifics to reproduce 80.1% IoU on validation, which you report. My initial model is **deeplab_resnet_init.ckpt**, which I reckon is a MS-COCO pre-trained model, but otherwise hasn't seen VOC. With *train.py* I get 82.6% / 74.8% on training / validation. *train_msc.py* yields 83.4% / 75.9%. I don't change anything set by default (i.e. 20000 iterations, learning rate, etc.). Is there any detail I missed?
Another question, did you also observe overfitting in your experiments?

Great job and thanks,
Nikita

**Comment by DrSleep** [comment]
it might be that the TF version is different. Otherwise, if you carefully followed README, you should have reached the numbers mentioned
